src/models/thelio-mira-r1.0/README.md

```diff
@@ -0,0 +1,81 @@
+- GPU options
+    - One or two GPUs, depending on configuration and power requirements (3x PCIe 3.0/4.0 x16 slots)
+    - Tested with the following GPUs:
+        - [NVIDIA GeForce GT 1030](https://www.nvidia.com/en-us/geforce/graphics-cards/gt-1030/)
+            - 1x HDMI 2.0b, 1x DVI-D (single link)
+        - [NVIDIA GeForce 2080 Super](https://www.nvidia.com/en-us/geforce/graphics-cards/rtx-2080-super/)
+            - 1x HDMI 2.0b, 3x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA GeForce RTX 3060 Ti](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3060-ti/)
+            - 1x HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA GeForce RTX 3070](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3070/)
+            - 1x HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA GeForce RTX 3080](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3080/)
+            - 1x HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA Quadro RTX 4000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-4000/)
+            - 3x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 5000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-5000/)
+            - 4x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 6000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-6000/)
+            - 4x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 8000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-8000/)
+            - 4x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [AMD Radeon RX 580](https://www.amd.com/en/products/graphics/radeon-rx-580)
+            - 2x DisplayPort 1.4, 2x HDMI 2.0b, 1x DVI-D (single link)
+        - [AMD Radeon RX 5700](https://www.amd.com/en/products/graphics/amd-radeon-rx-5700)
+            - 3x DisplayPort 1.4, 1x HDMI 2.0b
+- Memory
+    - Up to 128GB (4x32GB) quad-channel ECC Unbuffered DDR4 DIMMs @ 3200 MHz
+    - Tested with the following RAM modules (may ship with other tested modules):
+        - [Kingston HX432C16FB3/8](https://www.kingston.com/dataSheets/HX432C16FB3_8.pdf)
+        - [Kingston HX432C16FB4/64](https://www.kingston.com/dataSheets/HX432C16FB3K4_64.pdf)
+        - [Kingston HX432C16FB3K4/128](https://www.kingston.com/dataSheets/HX432C16FB3K4_128.pdf)
+        - [Micron MTA9ASF2G72AZ-3G2B1](https://www.micron.com/products/dram-modules/udimm/part-catalog/mta9asf2g72az-3g2/mta9asf2g72az-3g2b1)
+        - [Micron MTA18ASF4G72AZ-3G2B1](https://www.micron.com/products/dram-modules/udimm/part-catalog/mta18asf4g72az-3g2/mta18asf4g72az-3g2b1)
+- Networking
+    - 1x 2.5-Gigabit Ethernet port (Realtek RTL8125-CG 2.5G LAN)
+    - 1x 1-Gigabit Ethernet port (Intel I211-AT, 1 x Gigabit LAN Controller)
```

Comment:
I'm a little confused by how this is listed. I see that the motherboard manual lists:
![image](https://user-images.githubusercontent.com/7199422/110865467-e5566000-8280-11eb-9056-e8b49ec1ce2f.png)
What does `1 x Gigabit LAN Controller` mean on this line? It looks like the Realtek port uses its own controller, any objections to dropping this redundant detail so it looks like this instead?
```
- 1x 2.5-Gigabit Ethernet port (Realtek RTL8125-CG)
- 1x 1-Gigabit Ethernet port (Intel I211-AT)
```
Or am I reading this wrong, and you're trying to say that both ports use a single Intel controller?

Reply:
Yeah that's totally weird. I have no idea why I wrote it. We should definitely change it to the way you wrote it.